processor/src/test/java/org/mapstruct/ap/test/decorator/PersonMapper.java

```diff
@@ -33,4 +34,6 @@
     PersonDto personToPersonDto(Person person);
 
     AddressDto addressToAddressDto(Address address);
+
+    void updateAddressFromDto(AddressDto dto, @MappingTarget Address address);
```

Comment:
This should be covered by `shouldDelegateNonDecoratedMethodsToDefaultImplementation()` in the same test.